x-pack/test/security_solution_endpoint_api_int/apis/resolver/events.ts

```diff
@@ -41,107 +44,231 @@ export default function ({ getService }: FtrProviderContext) {
     ancestryArraySize: 2,
   };
 
-  describe('related events route', () => {
-    before(async () => {
-      await esArchiver.load('endpoint/resolver/api_feature');
-      resolverTrees = await resolver.createTrees(treeOptions);
-      // we only requested a single alert so there's only 1 tree
-      tree = resolverTrees.trees[0];
-    });
-    after(async () => {
-      await resolver.deleteData(resolverTrees);
-      await esArchiver.unload('endpoint/resolver/api_feature');
-    });
+  describe('event routes', () => {
+    describe('related events route', () => {
```

Comment:
These changes look worse than they are. This block was wrapped in the above `describe` block and intended.

Reply:
*indented